### PR TITLE
Details of transition rules

### LINKED ISSUE #144: Transition notes: update section 13.2
Introduce more clarity about the process of transitioning to a new Code of Practice, including matters such as dual running and availability of previous reports

### PR DESCRIPTION
**Pull request by tashalouizamc**
Clarifying dual running, availability of previous reports, transition timelines, etc.

### CODE REVIEW

File: source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst [hunk 1]
```diff
@@ -5,8 +5,22 @@
...
-* MUST support COUNTER reports that may include a range of months that span the transition period. If the new reporting service was deployed in August of 2024, a customer could request a report for January-December 2024 and receive a single report.
-* When it is not practical to support a single report with date ranges that span the transition period, the content provider MUST perform the transition on the first day of a month. If the new reporting service was deployed in August 2024, a customer wanting January-December 2024 usage would request January-July 2024 from the previous reporting service and August-December 2024 from the new reporting service. For clarity, a provider MUST NOT perform the transition mid-month such that the customer is required to run reports on both the old and new reporting services for the same month and merge and sum the results to obtain actual monthly usage.
+Availability of previous reports
+""""""""""""""""""""""""""""""""
+
+* Report providers MUST support COUNTER reports that may include a range of months that span the transition period.
```
Comment (berndoberknapp) [inline source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst]: The MUST here contradicts the relaxed rules in the next bullet points. Maybe this bullet point simply should be dropped?

File: source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst [hunk 2]
```diff
@@ -5,8 +5,22 @@
...
+* Report providers SHOULD offer a single report with date ranges that span the transition period. That is, if the new reporting service was deployed in August of 2024 a customer could request a report for January-December 2024 and receive a single report.
+* When it is not practical to support a single report with date ranges that span the transition period, the content provider SHOULD perform the transition on the first day of a month. If the new reporting service was deployed in August 2024, a customer wanting January-December 2024 usage would request January-July 2024 from the previous reporting service and August-December 2024 from the new reporting service. 
+
+Project COUNTER is aware that it is not always possible to transition on the first day of a month (e.g. where a platform is moving from one service provider to another). Where a report provider has no option but to perform the transition mid-month, such that the customer is required to run reports on both the old and new reporting services for the same month, the report provider MUST provide clear guidance about and merging and summing the results to obtain actual monthly usage.
```
Comment (berndoberknapp) [inline source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst]: "about merging" (remove "and").

File: source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst [hunk 3]
```diff
@@ -5,8 +5,22 @@
...
 
-* MUST continue to meet the requirement to offer valid COUNTER reports for the current year plus the prior 24 months (or from the date they first became compliant, whichever is later) via a web interface and via a SUSHI server.
-* MUST support COUNTER reports that may include a range of months that span the transition period. If the new reporting service was deployed in August of 2024, a customer could request a report for January-December 2024 and receive a single report.
-* When it is not practical to support a single report with date ranges that span the transition period, the content provider MUST perform the transition on the first day of a month. If the new reporting service was deployed in August 2024, a customer wanting January-December 2024 usage would request January-July 2024 from the previous reporting service and August-December 2024 from the new reporting service. For clarity, a provider MUST NOT perform the transition mid-month such that the customer is required to run reports on both the old and new reporting services for the same month and merge and sum the results to obtain actual monthly usage.
+Availability of previous reports
```
Comment (berndoberknapp) [inline source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst]: Subsection titles should be capitalized. Please include two empty lines before a subsection.

File: source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst [hunk 4]
```diff
@@ -5,8 +5,22 @@
...
+
+Project COUNTER is aware that it is not always possible to transition on the first day of a month (e.g. where a platform is moving from one service provider to another). Where a report provider has no option but to perform the transition mid-month, such that the customer is required to run reports on both the old and new reporting services for the same month, the report provider MUST provide clear guidance about and merging and summing the results to obtain actual monthly usage.
+
+Report providers MUST alert the COUNTER Project Director (tasha.mellins-cohen@counterusage.org) in advance of platforms transitioning to a new reporting service to ensure that the COUNTER Registry remains up-to-date (https://registry.projectcounter.org/). Transition details, including the date of transition, will be noted in the specific platform record and as a notification to report consumers.
+
+Dual running of reporting systems
```
Comment (berndoberknapp) [inline source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst]: Subsection titles should be capitalized. Please include two empty lines before a subsection.

File: source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst [hunk 5]
```diff
@@ -5,8 +5,22 @@
...
+"""""""""""""""""""""""""""""""""
+
+Report providers have two options in respect of delivering previous reports, where they are unable to offer a single report with date ranges that span the transition period. 
+    * Support two reporting systems, such that reports on usage dating prior to transition are held in the old reporting service. In this case report consumers will access post-transition reports on the new reporting service and pre-transition reports on the old reporting service.
+    * Support the pre-transition reports on the new reporting service.
```
Comment (berndoberknapp) [inline source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst]: There must be an empty line before a list. The bullet points should not be indented.

**Review by berndoberknapp — CHANGES_REQUESTED** [review]
Please see the inline comments.

**Comment by tashalouizamc** [comment]
New commit addressing comments, 2023-03-15

File: source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst [hunk 6]
```diff
@@ -5,8 +5,24 @@
...
+
+Availability of Previous Reports
+""""""""""""""""""""""""""""""""
+
+* Report providers SHOULD offer a single report with date ranges that span the transition period. That is, if the new reporting service was deployed in August of 2024 a customer could request a report for January-December 2024 and receive a single report.
+* When it is not practical to support a single report with date ranges that span the transition period, the content provider SHOULD perform the transition on the first day of a month. If the new reporting service was deployed in August 2024, a customer wanting January-December 2024 usage would request January-July 2024 from the previous reporting service and August-December 2024 from the new reporting service.
```
Comment (berndoberknapp) [inline source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst:15]: I would suggest to simply use paragraphs instead of bullet points.

**Review by berndoberknapp — APPROVED** [review]

File: source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst (hunk 6 above)
Comment (tashalouizamc) [reply source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst:15]: On the list for the copy-edit issue

**Pull request merged**